Makefile

```diff
@@ -1,58 +1,66 @@
-# Copyright 2023 K8sgpt AI. All rights reserved.
-# Use of this source code is governed by a MIT style
-# license that can be found in the LICENSE file.
+# Copyright 2023 The K8sGPT Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # ==============================================================================
 # define the default goal
 #
 ROOT_PACKAGE=github.com/k8sgpt-ai/k8sgpt
-
-SHELL := /bin/bash
-DIRS=$(shell ls)
-GO=go
+VERSION_PACKAGE=github.com/k8sgpt-ai/k8sgpt/version
```

Comment:
The version is currently stored here [.release-please-manifest.json](https://github.com/k8sgpt-ai/k8sgpt/blob/main/.release-please-manifest.json) I would advise against us keeping multiple sources of truth